pyscf/nao/gw.py

```diff
@@ -48,12 +48,12 @@ def __init__(self, **kw):
     else :
       self.nvrt = array([min(6,j) for j in self.norbs-nocc_0t])
 
-    if self.verbosity>0: print(__name__, 'nocc =', self.nocc, 'nvrt =', self.nvrt)
+    if self.verbosity>0: print(__name__,'\t\t====> Number of ocupied states are = {}, Number of virtual states = {}'.format(self.nocc, self.nvrt))
```

Comment:
Well, here you rename the input variables, reporting them as "Number of ocupied states are". nocc and nvrt is descriptive for the user and gives the clue on the names of optional arguments.